### ISSUE TITLE
[Bug]: Syntax errors in collecting_event.rb preventing rails server from being booted

### ISSUE THREAD

**Issue by sdjbrown** [SpeciesFileGroup/taxonworks#3381]
### Steps to reproduce the bug

```bash
1. When booting an up-to-date instance of TaxonWorks using `bundle exec rails s`, the command fails with a series of syntax errors in `collecting_event.rb`
2. Errors are also returned when simultaneously running `bin/webpacker-dev-server`, but these do not seem to be fatal
```

### Screenshot

Error messages from running `bundle exec rails s`

PATH$ bundle exec rails s
HOME/.rvm/rubies/ruby-3.0.2/lib/ruby/3.0.0/digest.rb:6: warning: already initialized constant Digest::REQUIRE_MUTEX
HOME/.rvm/gems/ruby-3.0.2/gems/digest-3.1.0/lib/digest.rb:20: warning: previous definition of REQUIRE_MUTEX was here
=> Booting Puma
=> Rails 6.1.7.3 application starting in development 
=> Run `bin/rails server --help` for more startup options
Exiting
HOME/.rvm/gems/ruby-3.0.2/gems/activesupport-6.1.7.3/lib/active_support/dependencies.rb:523:in `load': PATH/app/models/collecting_event.rb:329: syntax error, unexpected '}' (SyntaxError)
...ection_objects: { project_id: } ) }
...                              ^
PATH/app/models/collecting_event.rb:338: syntax error, unexpected tLAMBEG
      h = {
          ^
PATH/app/models/collecting_event.rb:346: syntax error, unexpected ','
...lectingEvent.where(project_id:, updated_by_id: user_id, crea...
...                              ^
PATH/app/models/collecting_event.rb:346: syntax error, unexpected ')', expecting `end'
...s.ago..Time.now).limit(5).to_a).uniq,
...                              ^
PATH/app/models/collecting_event.rb:348: syntax error, unexpected '}', expecting `end'
      }
      ^
PATH/app/models/collecting_event.rb:449: syntax error, unexpected ')'
      .where.not(id:).distinct
                    ^
PATH/app/models/collecting_event.rb:456: syntax error, unexpected `do' for lambda
...ngEvent.core_attributes.each do |a|
...                             ^~
PATH/app/models/collecting_event.rb:457: void value expression
      return true if self.send(a).present?
      ^~~~~~
PATH/app/models/collecting_event.rb:457: syntax error, unexpected `true', expecting `end'
      return true if self.send(a).present?
             ^~~~
PATH/app/models/collecting_event.rb:461: syntax error, unexpected `end', expecting '}'
  end
  ^~~
PATH/app/models/collecting_event.rb:567: syntax error, unexpected '}'
..._event_id: id.to_s, no_cached:}
...                              ^
PATH/app/models/collecting_event.rb:568: syntax error, unexpected ')'
...e!(by: creator.id, project_id:) if reference_self
...                              ^
PATH/app/models/collecting_event.rb:589: syntax error, unexpected ')'
...     GeographicItem.new(point:)
...                              ^
PATH/app/models/collecting_event.rb:613: syntax error, unexpected ')'
...Item.where(geographic_area_id:)
...                              ^
PATH/app/models/collecting_event.rb:1154: syntax error, unexpected `end', expecting '}'
  end
  ^~~
        from HOME/.rvm/gems/ruby-3.0.2/gems/activesupport-6.1.7.3/lib/active_support/dependencies.rb:523:in `block in load_file'
        from HOME/.rvm/gems/ruby-3.0.2/gems/activesupport-6.1.7.3/lib/active_support/dependencies.rb:715:in `new_constants_in'
        from HOME/.rvm/gems/ruby-3.0.2/gems/activesupport-6.1.7.3/lib/active_support/dependencies.rb:522:in `load_file'
        from HOME/.rvm/gems/ruby-3.0.2/gems/activesupport-6.1.7.3/lib/active_support/dependencies.rb:419:in `block in require_or_load'
        from HOME/.rvm/gems/ruby-3.0.2/gems/activesupport-6.1.7.3/lib/active_support/dependencies.rb:39:in `block in load_interlock'
        from HOME/.rvm/gems/ruby-3.0.2/gems/activesupport-6.1.7.3/lib/active_support/dependencies/interlock.rb:14:in `block in loading'
        from HOME/.rvm/gems/ruby-3.0.2/gems/activesupport-6.1.7.3/lib/active_support/concurrency/share_lock.rb:151:in `exclusive'
        from HOME/.rvm/gems/ruby-3.0.2/gems/activesupport-6.1.7.3/lib/active_support/dependencies/interlock.rb:13:in `loading'
        from HOME/.rvm/gems/ruby-3.0.2/gems/activesupport-6.1.7.3/lib/active_support/dependencies.rb:39:in `load_interlock'
        from HOME/.rvm/gems/ruby-3.0.2/gems/activesupport-6.1.7.3/lib/active_support/dependencies.rb:402:in `require_or_load'
        from HOME/.rvm/gems/ruby-3.0.2/gems/activesupport-6.1.7.3/lib/active_support/dependencies.rb:375:in `depend_on'
        from HOME/.rvm/gems/ruby-3.0.2/gems/activesupport-6.1.7.3/lib/active_support/dependencies.rb:288:in `require_dependency'
        from PATH/lib/application_enumeration.rb:1:in `block in <top (required)>'
        from PATH/lib/application_enumeration.rb:1:in `each'
        from PATH/lib/application_enumeration.rb:1:in `<top (required)>'
        from HOME/.rvm/gems/ruby-3.0.2/gems/activesupport-6.1.7.3/lib/active_support/dependencies.rb:523:in `load'
        from HOME/.rvm/gems/ruby-3.0.2/gems/activesupport-6.1.7.3/lib/active_support/dependencies.rb:523:in `block in load_file'
        from HOME/.rvm/gems/ruby-3.0.2/gems/activesupport-6.1.7.3/lib/active_support/dependencies.rb:715:in `new_constants_in'
        from HOME/.rvm/gems/ruby-3.0.2/gems/activesupport-6.1.7.3/lib/active_support/dependencies.rb:522:in `load_file'
        from HOME/.rvm/gems/ruby-3.0.2/gems/activesupport-6.1.7.3/lib/active_support/dependencies.rb:419:in `block in require_or_load'
        from HOME/.rvm/gems/ruby-3.0.2/gems/activesupport-6.1.7.3/lib/active_support/dependencies.rb:39:in `block in load_interlock'
        from HOME/.rvm/gems/ruby-3.0.2/gems/activesupport-6.1.7.3/lib/active_support/dependencies/interlock.rb:14:in `block in loading'
        from HOME/.rvm/gems/ruby-3.0.2/gems/activesupport-6.1.7.3/lib/active_support/concurrency/share_lock.rb:151:in `exclusive'
        from HOME/.rvm/gems/ruby-3.0.2/gems/activesupport-6.1.7.3/lib/active_support/dependencies/interlock.rb:13:in `loading'
        from HOME/.rvm/gems/ruby-3.0.2/gems/activesupport-6.1.7.3/lib/active_support/dependencies.rb:39:in `load_interlock'
        from HOME/.rvm/gems/ruby-3.0.2/gems/activesupport-6.1.7.3/lib/active_support/dependencies.rb:402:in `require_or_load'
        from HOME/.rvm/gems/ruby-3.0.2/gems/activesupport-6.1.7.3/lib/active_support/dependencies.rb:558:in `load_missing_constant'
        from HOME/.rvm/gems/ruby-3.0.2/gems/activesupport-6.1.7.3/lib/active_support/dependencies.rb:213:in `const_missing'
        from PATH/config/initializers/constants/model/is_data.rb:4:in `<top (required)>'
        from HOME/.rvm/gems/ruby-3.0.2/gems/activesupport-6.1.7.3/lib/active_support/dependencies.rb:326:in `load'
        from HOME/.rvm/gems/ruby-3.0.2/gems/activesupport-6.1.7.3/lib/active_support/dependencies.rb:326:in `block in load'
        from HOME/.rvm/gems/ruby-3.0.2/gems/activesupport-6.1.7.3/lib/active_support/dependencies.rb:299:in `load_dependency'
        from HOME/.rvm/gems/ruby-3.0.2/gems/activesupport-6.1.7.3/lib/active_support/dependencies.rb:326:in `load'
        from HOME/.rvm/gems/ruby-3.0.2/gems/railties-6.1.7.3/lib/rails/engine.rb:681:in `block in load_config_initializer'
        from HOME/.rvm/gems/ruby-3.0.2/gems/activesupport-6.1.7.3/lib/active_support/notifications.rb:205:in `instrument'
        from HOME/.rvm/gems/ruby-3.0.2/gems/railties-6.1.7.3/lib/rails/engine.rb:680:in `load_config_initializer'
        from HOME/.rvm/gems/ruby-3.0.2/gems/railties-6.1.7.3/lib/rails/engine.rb:634:in `block (2 levels) in <class:Engine>'
        from HOME/.rvm/gems/ruby-3.0.2/gems/railties-6.1.7.3/lib/rails/engine.rb:633:in `each'
        from HOME/.rvm/gems/ruby-3.0.2/gems/railties-6.1.7.3/lib/rails/engine.rb:633:in `block in <class:Engine>'
        from HOME/.rvm/gems/ruby-3.0.2/gems/railties-6.1.7.3/lib/rails/initializable.rb:32:in `instance_exec'
        from HOME/.rvm/gems/ruby-3.0.2/gems/railties-6.1.7.3/lib/rails/initializable.rb:32:in `run'
        from HOME/.rvm/gems/ruby-3.0.2/gems/railties-6.1.7.3/lib/rails/initializable.rb:61:in `block in run_initializers'
        from HOME/.rvm/rubies/ruby-3.0.2/lib/ruby/3.0.0/tsort.rb:228:in `block in tsort_each'
        from HOME/.rvm/rubies/ruby-3.0.2/lib/ruby/3.0.0/tsort.rb:350:in `block (2 levels) in each_strongly_connected_component'
        from HOME/.rvm/rubies/ruby-3.0.2/lib/ruby/3.0.0/tsort.rb:422:in `block (2 levels) in each_strongly_connected_component_from'
        from HOME/.rvm/rubies/ruby-3.0.2/lib/ruby/3.0.0/tsort.rb:431:in `each_strongly_connected_component_from'
        from HOME/.rvm/rubies/ruby-3.0.2/lib/ruby/3.0.0/tsort.rb:421:in `block in each_strongly_connected_component_from'
        from HOME/.rvm/gems/ruby-3.0.2/gems/railties-6.1.7.3/lib/rails/initializable.rb:50:in `each'
        from HOME/.rvm/gems/ruby-3.0.2/gems/railties-6.1.7.3/lib/rails/initializable.rb:50:in `tsort_each_child'
        from HOME/.rvm/rubies/ruby-3.0.2/lib/ruby/3.0.0/tsort.rb:415:in `call'
        from HOME/.rvm/rubies/ruby-3.0.2/lib/ruby/3.0.0/tsort.rb:415:in `each_strongly_connected_component_from'
        from HOME/.rvm/rubies/ruby-3.0.2/lib/ruby/3.0.0/tsort.rb:349:in `block in each_strongly_connected_component'
        from HOME/.rvm/rubies/ruby-3.0.2/lib/ruby/3.0.0/tsort.rb:347:in `each'
        from HOME/.rvm/rubies/ruby-3.0.2/lib/ruby/3.0.0/tsort.rb:347:in `call'
        from HOME/.rvm/rubies/ruby-3.0.2/lib/ruby/3.0.0/tsort.rb:347:in `each_strongly_connected_component'
        from HOME/.rvm/rubies/ruby-3.0.2/lib/ruby/3.0.0/tsort.rb:226:in `tsort_each'
        from HOME/.rvm/rubies/ruby-3.0.2/lib/ruby/3.0.0/tsort.rb:205:in `tsort_each'
        from HOME/.rvm/gems/ruby-3.0.2/gems/railties-6.1.7.3/lib/rails/initializable.rb:60:in `run_initializers'
        from HOME/.rvm/gems/ruby-3.0.2/gems/railties-6.1.7.3/lib/rails/application.rb:391:in `initialize!'
        from PATH/config/environment.rb:5:in `<top (required)>'
        from config.ru:3:in `require_relative'
        from config.ru:3:in `block in <main>'
        from HOME/.rvm/gems/ruby-3.0.2/gems/rack-2.2.6.4/lib/rack/builder.rb:116:in `eval'
        from HOME/.rvm/gems/ruby-3.0.2/gems/rack-2.2.6.4/lib/rack/builder.rb:116:in `new_from_string'
        from HOME/.rvm/gems/ruby-3.0.2/gems/rack-2.2.6.4/lib/rack/builder.rb:105:in `load_file'
        from HOME/.rvm/gems/ruby-3.0.2/gems/rack-2.2.6.4/lib/rack/builder.rb:66:in `parse_file'
        from HOME/.rvm/gems/ruby-3.0.2/gems/rack-2.2.6.4/lib/rack/server.rb:349:in `build_app_and_options_from_config'
        from HOME/.rvm/gems/ruby-3.0.2/gems/rack-2.2.6.4/lib/rack/server.rb:249:in `app'
        from HOME/.rvm/gems/ruby-3.0.2/gems/rack-2.2.6.4/lib/rack/server.rb:422:in `wrapped_app'
        from HOME/.rvm/gems/ruby-3.0.2/gems/railties-6.1.7.3/lib/rails/commands/server/server_command.rb:77:in `log_to_stdout'
        from HOME/.rvm/gems/ruby-3.0.2/gems/railties-6.1.7.3/lib/rails/commands/server/server_command.rb:37:in `start'
        from HOME/.rvm/gems/ruby-3.0.2/gems/railties-6.1.7.3/lib/rails/commands/server/server_command.rb:144:in `block in perform'
        from <internal:kernel>:90:in `tap'
        from HOME/.rvm/gems/ruby-3.0.2/gems/railties-6.1.7.3/lib/rails/commands/server/server_command.rb:135:in `perform'
        from HOME/.rvm/gems/ruby-3.0.2/gems/thor-1.2.1/lib/thor/command.rb:27:in `run'
        from HOME/.rvm/gems/ruby-3.0.2/gems/thor-1.2.1/lib/thor/invocation.rb:127:in `invoke_command'
        from HOME/.rvm/gems/ruby-3.0.2/gems/thor-1.2.1/lib/thor.rb:392:in `dispatch'
        from HOME/.rvm/gems/ruby-3.0.2/gems/railties-6.1.7.3/lib/rails/command/base.rb:69:in `perform'
        from HOME/.rvm/gems/ruby-3.0.2/gems/railties-6.1.7.3/lib/rails/command.rb:48:in `invoke'
        from HOME/.rvm/gems/ruby-3.0.2/gems/railties-6.1.7.3/lib/rails/commands.rb:18:in `<top (required)>'
        from bin/rails:4:in `require'
        from bin/rails:4:in `<main>'

### Expected behavior

When running `bundle exec rails s`, I expect the server to boot successfully, allowing me to access http://127.0.0.1:3000/ from my browser.

### Additional Screenshots

Error messages from running `bin/webpacker-dev-server`

PATH$ bin/webpacker-dev-server
HOME/.rvm/rubies/ruby-3.0.2/lib/ruby/3.0.0/digest.rb:6: warning: already initialized constant Digest::REQUIRE_MUTEX
HOME/.rvm/gems/ruby-3.0.2/gems/digest-3.1.0/lib/digest.rb:20: warning: previous definition of REQUIRE_MUTEX was here
<i> [webpack-dev-server] Project is running at:
<i> [webpack-dev-server] Loopback: http://localhost:3035/, http://127.0.0.1:3035/
<i> [webpack-dev-server] Content not from webpack is served from 'PATH/public' directory
<i> [webpack-dev-server] 404s will fallback to '/index.html'
assets by path static/ 130 KiB
  assets by chunk 121 KiB (auxiliary name: application) 16 assets
  assets by chunk 8.84 KiB (auxiliary id hint: vendors)
    asset static/pdfjs-dist/web/images/loading-icon-ff2dfa442d903d9d5e30.gif 2.49 KiB [emitted] [immutable] [from: node_modules/pdfjs-dist/web/images/loading-icon.gif] (auxiliary id hint: vendors)
    asset static/leaflet/dist/images/marker-icon-2x-680f69f3c2e6b90c1812.png 2.41 KiB [emitted] [immutable] [from: node_modules/leaflet/dist/images/marker-icon-2x.png] (auxiliary id hint: vendors)
    + 4 assets
assets by path js/*.js 25.1 MiB
  assets by chunk 7.68 MiB (id hint: vendors)
    asset js/vendors-node_modules_geoman-io_leaflet-geoman-free_dist_leaflet-geoman_min_js-node_modules_sf-3667ac.js 5.8 MiB [emitted] (id hint: vendors) 1 related asset
    asset js/vendors-node_modules_pdfjs-dist_build_pdf_worker_js.chunk.js 1.88 MiB [emitted] (id hint: vendors) 1 related asset
  + 5 assets
assets by path css/*.css 511 KiB
  asset css/application.css 424 KiB [emitted] (name: application) 1 related asset
  asset css/vendors-node_modules_geoman-io_leaflet-geoman-free_dist_leaflet-geoman_min_js-node_modules_sf-3667ac.css 86.8 KiB [emitted] (id hint: vendors) 1 related asset                                                                                                  
asset manifest.json 6.84 KiB [emitted]

ERROR in ./app/javascript/vue/tasks/biological_associations/biological_associations_graph/constants/networkConfig.js 1:0-48
Module not found: Error: Can't resolve 'v-network-graph' in 'PATH/app/javascript/vue/tasks/biological_associations/biological_associations_graph/constants'
resolve 'v-network-graph' in 'PATH/app/javascript/vue/tasks/biological_associations/biological_associations_graph/constants'
  Parsed request is a module
  using description file: PATH/package.json (relative path: ./app/javascript/vue/tasks/biological_associations/biological_associations_graph/constants)
    Field 'browser' doesn't contain a valid alias configuration
    resolve as module
      looking for modules in PATH/app/javascript
        single file module
          using description file: PATH/package.json (relative path: ./app/javascript/v-network-graph)
            no extension
              Field 'browser' doesn't contain a valid alias configuration
              PATH/app/javascript/v-network-graph doesn't exist
            .css
              Field 'browser' doesn't contain a valid alias configuration
              PATH/app/javascript/v-network-graph.css doesn't exist
            .vue
              Field 'browser' doesn't contain a valid alias configuration
              PATH/app/javascript/v-network-graph.vue doesn't exist
            .js
              Field 'browser' doesn't contain a valid alias configuration
              PATH/app/javascript/v-network-graph.js doesn't exist
            .jsx
              Field 'browser' doesn't contain a valid alias configuration
              PATH/app/javascript/v-network-graph.jsx doesn't exist
            .mjs
              Field 'browser' doesn't contain a valid alias configuration
              PATH/app/javascript/v-network-graph.mjs doesn't exist
            .ts
              Field 'browser' doesn't contain a valid alias configuration
              PATH/app/javascript/v-network-graph.ts doesn't exist
            .tsx
              Field 'browser' doesn't contain a valid alias configuration
              PATH/app/javascript/v-network-graph.tsx doesn't exist
            .coffee
              Field 'browser' doesn't contain a valid alias configuration
              PATH/app/javascript/v-network-graph.coffee doesn't exist
        PATH/app/javascript/v-network-graph doesn't exist
      looking for modules in PATH/app/javascript/vue
        single file module
          using description file: PATH/package.json (relative path: ./app/javascript/vue/v-network-graph)
            no extension
              Field 'browser' doesn't contain a valid alias configuration
              PATH/app/javascript/vue/v-network-graph doesn't exist
            .css
              Field 'browser' doesn't contain a valid alias configuration
              PATH/app/javascript/vue/v-network-graph.css doesn't exist
            .vue
              Field 'browser' doesn't contain a valid alias configuration
              PATH/app/javascript/vue/v-network-graph.vue doesn't exist
            .js
              Field 'browser' doesn't contain a valid alias configuration
              PATH/app/javascript/vue/v-network-graph.js doesn't exist
            .jsx
              Field 'browser' doesn't contain a valid alias configuration
              PATH/app/javascript/vue/v-network-graph.jsx doesn't exist
            .mjs
              Field 'browser' doesn't contain a valid alias configuration
              PATH/app/javascript/vue/v-network-graph.mjs doesn't exist
            .ts
              Field 'browser' doesn't contain a valid alias configuration
              PATH/app/javascript/vue/v-network-graph.ts doesn't exist
            .tsx
              Field 'browser' doesn't contain a valid alias configuration
              PATH/app/javascript/vue/v-network-graph.tsx doesn't exist
            .coffee
              Field 'browser' doesn't contain a valid alias configuration
              PATH/app/javascript/vue/v-network-graph.coffee doesn't exist
        PATH/app/javascript/vue/v-network-graph doesn't exist
      PATH/app/javascript/vue/tasks/biological_associations/biological_associations_graph/constants/node_modules doesn't exist or is not a directory                                                                                           
      PATH/app/javascript/vue/tasks/biological_associations/biological_associations_graph/node_modules doesn't exist or is not a directory                                                                                                     
      PATH/app/javascript/vue/tasks/biological_associations/node_modules doesn't exist or is not a directory
      PATH/app/javascript/vue/tasks/node_modules doesn't exist or is not a directory
      PATH/app/javascript/vue/node_modules doesn't exist or is not a directory
      PATH/app/javascript/node_modules doesn't exist or is not a directory
      PATH/app/node_modules doesn't exist or is not a directory
      looking for modules in PATH/node_modules
        single file module
          using description file: PATH/package.json (relative path: ./node_modules/v-network-graph)
            no extension
              Field 'browser' doesn't contain a valid alias configuration
              PATH/node_modules/v-network-graph doesn't exist
            .css
              Field 'browser' doesn't contain a valid alias configuration
              PATH/node_modules/v-network-graph.css doesn't exist
            .vue
              Field 'browser' doesn't contain a valid alias configuration
              PATH/node_modules/v-network-graph.vue doesn't exist
            .js
              Field 'browser' doesn't contain a valid alias configuration
              PATH/node_modules/v-network-graph.js doesn't exist
            .jsx
              Field 'browser' doesn't contain a valid alias configuration
              PATH/node_modules/v-network-graph.jsx doesn't exist
            .mjs
              Field 'browser' doesn't contain a valid alias configuration
              PATH/node_modules/v-network-graph.mjs doesn't exist
            .ts
              Field 'browser' doesn't contain a valid alias configuration
              PATH/node_modules/v-network-graph.ts doesn't exist
            .tsx
              Field 'browser' doesn't contain a valid alias configuration
              PATH/node_modules/v-network-graph.tsx doesn't exist
            .coffee
              Field 'browser' doesn't contain a valid alias configuration
              PATH/node_modules/v-network-graph.coffee doesn't exist
        PATH/node_modules/v-network-graph doesn't exist
      HOME/Documents/TW/node_modules doesn't exist or is not a directory
      HOME/Documents/node_modules doesn't exist or is not a directory
      HOME/node_modules doesn't exist or is not a directory
      /home/node_modules doesn't exist or is not a directory
      /node_modules doesn't exist or is not a directory

ERROR in ./app/javascript/vue/tasks/biological_associations/biological_associations_graph/main.js 3:0-44
Module not found: Error: Can't resolve 'v-network-graph' in 'PATH/app/javascript/vue/tasks/biological_associations/biological_associations_graph'
resolve 'v-network-graph' in 'PATH/app/javascript/vue/tasks/biological_associations/biological_associations_graph'
  Parsed request is a module
  using description file: PATH/package.json (relative path: ./app/javascript/vue/tasks/biological_associations/biological_associations_graph)
    Field 'browser' doesn't contain a valid alias configuration
    resolve as module
      looking for modules in PATH/app/javascript
        single file module
          using description file: PATH/package.json (relative path: ./app/javascript/v-network-graph)
            no extension
              Field 'browser' doesn't contain a valid alias configuration
              PATH/app/javascript/v-network-graph doesn't exist
            .css
              Field 'browser' doesn't contain a valid alias configuration
              PATH/app/javascript/v-network-graph.css doesn't exist
            .vue
              Field 'browser' doesn't contain a valid alias configuration
              PATH/app/javascript/v-network-graph.vue doesn't exist
            .js
              Field 'browser' doesn't contain a valid alias configuration
              PATH/app/javascript/v-network-graph.js doesn't exist
            .jsx
              Field 'browser' doesn't contain a valid alias configuration
              PATH/app/javascript/v-network-graph.jsx doesn't exist
            .mjs
              Field 'browser' doesn't contain a valid alias configuration
              PATH/app/javascript/v-network-graph.mjs doesn't exist
            .ts
              Field 'browser' doesn't contain a valid alias configuration
              PATH/app/javascript/v-network-graph.ts doesn't exist
            .tsx
              Field 'browser' doesn't contain a valid alias configuration
              PATH/app/javascript/v-network-graph.tsx doesn't exist
            .coffee
              Field 'browser' doesn't contain a valid alias configuration
              PATH/app/javascript/v-network-graph.coffee doesn't exist
        PATH/app/javascript/v-network-graph doesn't exist
      looking for modules in PATH/app/javascript/vue
        single file module
          using description file: PATH/package.json (relative path: ./app/javascript/vue/v-network-graph)
            no extension
              Field 'browser' doesn't contain a valid alias configuration
              PATH/app/javascript/vue/v-network-graph doesn't exist
            .css
              Field 'browser' doesn't contain a valid alias configuration
              PATH/app/javascript/vue/v-network-graph.css doesn't exist
            .vue
              Field 'browser' doesn't contain a valid alias configuration
              PATH/app/javascript/vue/v-network-graph.vue doesn't exist
            .js
              Field 'browser' doesn't contain a valid alias configuration
              PATH/app/javascript/vue/v-network-graph.js doesn't exist
            .jsx
              Field 'browser' doesn't contain a valid alias configuration
              PATH/app/javascript/vue/v-network-graph.jsx doesn't exist
            .mjs
              Field 'browser' doesn't contain a valid alias configuration
              PATH/app/javascript/vue/v-network-graph.mjs doesn't exist
            .ts
              Field 'browser' doesn't contain a valid alias configuration
              PATH/app/javascript/vue/v-network-graph.ts doesn't exist
            .tsx
              Field 'browser' doesn't contain a valid alias configuration
              PATH/app/javascript/vue/v-network-graph.tsx doesn't exist
            .coffee
              Field 'browser' doesn't contain a valid alias configuration
              PATH/app/javascript/vue/v-network-graph.coffee doesn't exist
        PATH/app/javascript/vue/v-network-graph doesn't exist
      PATH/app/javascript/vue/tasks/biological_associations/biological_associations_graph/node_modules doesn't exist or is not a directory                                                                                                     
      PATH/app/javascript/vue/tasks/biological_associations/node_modules doesn't exist or is not a directory
      PATH/app/javascript/vue/tasks/node_modules doesn't exist or is not a directory
      PATH/app/javascript/vue/node_modules doesn't exist or is not a directory
      PATH/app/javascript/node_modules doesn't exist or is not a directory
      PATH/app/node_modules doesn't exist or is not a directory
      looking for modules in PATH/node_modules
        single file module
          using description file: PATH/package.json (relative path: ./node_modules/v-network-graph)
            no extension
              Field 'browser' doesn't contain a valid alias configuration
              PATH/node_modules/v-network-graph doesn't exist
            .css
              Field 'browser' doesn't contain a valid alias configuration
              PATH/node_modules/v-network-graph.css doesn't exist
            .vue
              Field 'browser' doesn't contain a valid alias configuration
              PATH/node_modules/v-network-graph.vue doesn't exist
            .js
              Field 'browser' doesn't contain a valid alias configuration
              PATH/node_modules/v-network-graph.js doesn't exist
            .jsx
              Field 'browser' doesn't contain a valid alias configuration
              PATH/node_modules/v-network-graph.jsx doesn't exist
            .mjs
              Field 'browser' doesn't contain a valid alias configuration
              PATH/node_modules/v-network-graph.mjs doesn't exist
            .ts
              Field 'browser' doesn't contain a valid alias configuration
              PATH/node_modules/v-network-graph.ts doesn't exist
            .tsx
              Field 'browser' doesn't contain a valid alias configuration
              PATH/node_modules/v-network-graph.tsx doesn't exist
            .coffee
              Field 'browser' doesn't contain a valid alias configuration
              PATH/node_modules/v-network-graph.coffee doesn't exist
        PATH/node_modules/v-network-graph doesn't exist
      HOME/Documents/TW/node_modules doesn't exist or is not a directory
      HOME/Documents/node_modules doesn't exist or is not a directory
      HOME/node_modules doesn't exist or is not a directory
      /home/node_modules doesn't exist or is not a directory
      /node_modules doesn't exist or is not a directory

ERROR in ./app/javascript/vue/tasks/biological_associations/biological_associations_graph/main.js 4:0-39
Module not found: Error: Can't resolve 'v-network-graph/lib/style.css' in 'PATH/app/javascript/vue/tasks/biological_associations/biological_associations_graph'
resolve 'v-network-graph/lib/style.css' in 'PATH/app/javascript/vue/tasks/biological_associations/biological_associations_graph'
  Parsed request is a module
  using description file: PATH/package.json (relative path: ./app/javascript/vue/tasks/biological_associations/biological_associations_graph)
    Field 'browser' doesn't contain a valid alias configuration
    resolve as module
      looking for modules in PATH/app/javascript
        PATH/app/javascript/v-network-graph doesn't exist
      looking for modules in PATH/app/javascript/vue
        PATH/app/javascript/vue/v-network-graph doesn't exist
      PATH/app/javascript/vue/tasks/biological_associations/biological_associations_graph/node_modules doesn't exist or is not a directory                                                                                                     
      PATH/app/javascript/vue/tasks/biological_associations/node_modules doesn't exist or is not a directory
      PATH/app/javascript/vue/tasks/node_modules doesn't exist or is not a directory
      PATH/app/javascript/vue/node_modules doesn't exist or is not a directory
      PATH/app/javascript/node_modules doesn't exist or is not a directory
      PATH/app/node_modules doesn't exist or is not a directory
      looking for modules in PATH/node_modules
        PATH/node_modules/v-network-graph doesn't exist
      HOME/Documents/TW/node_modules doesn't exist or is not a directory
      HOME/Documents/node_modules doesn't exist or is not a directory
      HOME/node_modules doesn't exist or is not a directory
      /home/node_modules doesn't exist or is not a directory
      /node_modules doesn't exist or is not a directory

webpack 5.75.0 compiled with 3 errors in 107555 ms

### Environment

Development (native)

### Version

commit 78309a4

### Browser Used

Firefox 112.0 (64-bit)

**Comment by sdjbrown** [comment]
Rolling back to commit 65d6a4b restores the desired behaviour.

This error was introduced in commit 07c4d19

**Comment by sdjbrown** [comment]
NB: The errors from running `bin/webpacker-dev-server` were fixed by installing `v-network-graph` with `npm install v-network-graph`

Sorry for mudding the waters by including the details relating to that aspect of things.

**Comment by LocoDelAssembly** [comment]
I cannot reproduce with `development` branch nor by checking out `78309a4bfb`.

What is your output for `bundle exec ruby -v` and `npm version`?

**Comment by jlpereira** [comment]
@sdjbrown run `bundle install` and `npm install`

v-netwrok-graph is already in package.json, sometimes we add new packages and update others. You have to install them to avoid these issues

**Comment by mjy** [comment]
@sdjbrown what version of Ruby are you using?

**Comment by sdjbrown** [comment]
$bundle exec ruby -v
ruby 3.0.2p107 (2021-07-07 revision 0db68f0233) [x86_64-linux]

$ npm version
{
  app: '0.1.0',
  npm: '6.14.18',
  ares: '1.18.1',
  brotli: '1.0.9',
  cldr: '40.0',
  icu: '70.1',
  llhttp: '2.1.6',
  modules: '83',
  napi: '8',
  nghttp2: '1.42.0',
  node: '14.21.3',
  openssl: '1.1.1t',
  tz: '2022f',
  unicode: '14.0',
  uv: '1.42.0',
  v8: '8.4.371.23-node.88',
  zlib: '1.2.11'
}

**Comment by LocoDelAssembly** [comment]
I can reproduce with version `3.0.2`. Using `param: ` without supplying a value does not and expect it to resolve to a reference to `param` variable doesn't work in Ruby versions lower than 3.1.

@mjy if we are going to keep using that syntax then I have to update the Gemfile to require 3.1 minimum.

**Comment by sdjbrown** [comment]
Issue resolved by `rvm install 3.2.1` followed by `bundle install`.
